docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -489,80 +612,63 @@
     "\n",
-    "| Ansatz                | Preparation method    |   Ansatz Setting Default  | Note                                                  |\n",
-    "| --------------------- | --------------------- | ------------------------- | ----------------------------------------------------- |\n",
-    "| Double Excitation     | CCSD                  | `{n_amplitudes: 10}`          | Lets you optimize from the result in the last example    |\n",
-    "| UCCSD                 | CCSD (default), Random          | `{use_singles: True, n_trotter: 1, reduce_parameter: True}` | Not recommended for systems with eight or more qubits      |\n",
-    "| KuCJ                  | Random                | `{k: 1}`                          | Recommended for system up to 16 qubits                 |\n",
-    "| KUpCCGSD              | Random                | `{k: 1, n_trotter: 1, reduce_parameter: False} `                         |                                                       |\n",
-    "| Hardware Efficient    | Random                | `{n_layers: depend on qubit size}        `                  |                                                       |\n",
+    "The JSON dictionary used to run this algorithm is similar to that of QSCI. It only contains one additional field, \"max_iter\", which lets you limit the resource usage. The default is 2000 iterations.\n",
     "\n",
-    "If you don't specify the preparation method and the ansatz setting, default values are used. For this algorithm, the QSCI energy is used as the cost function for the COBYLA optimizer. In each iteration, the circuit parameters are updated and the QSCI energy is evaluated with the circuit carrying the new set of circuit parameters.\n",
-    "\n",
-    "The JSON dictionary used to run this algorithm is similar to that of QSCI. It contains one additional field, \"max_iter\", for limiting the resource usage. The default is 2000 iterations.\n",
-    "\n",
-    "The follwing is an example of running this algorithm with the 1-uCJ ansatz:"
+    "Example of running this algorithm with the 1-uCJ ansatz:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 86,
-   "id": "2ba81e9e-1dfe-4a74-88ba-7a1a0fc0338f",
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "opt_qsci_1ucj_json = {\n",
-    "    \"mole\": mole,\n",
     "    \"ansatz\": \"KuCJ\",\n",
     "    \"state_prep_method\": \"RANDOM\",  # this is optional and defaulted to \"RANDOM\" for \"KuCJ\".\n",
     "    \"ansatz_setting\": {\"k\": 1},     # this is optional and defaulted to the table above.\n",
```

Comment:
```suggestion
    "    \"ansatz_setting\": {\"k\": 1},     # This is optional with the default described in the previous table.\n",
```

Reply:
If the table isn't directly above, this should state the name of the table so people can find it.